src/test/java/core/basesyntax/service/ReportServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        reportService = new ReportServiceImpl(new StorageDaoImpl());
+    }
+
+    @Test
+    public void createReport_Okay() {
+        Map<String, Integer> calculationMap = Storage.getCalculationMap();
```

Comment:
clear storage after the test class. other tests may need it

Reply:
> clear storage after the test class. other tests may need it

fixed)